agent/fw_symfony4.c

```diff
@@ -39,8 +39,17 @@ NR_PHP_WRAPPER(nr_symfony4_exception) {
     goto end;
   }
 
-  /* Get the exception from the event. */
-  exception = nr_php_call(event, "getException");
+  /*
+   * Get the exception from the event.
+   * Firstly, we check if ExceptionEvent is available - if yes, that means we are using Symfony 5
+   */
+  if (nr_php_object_instanceof_class(
+        event, "Symfony\\Component\\HttpKernel\\Event\\ExceptionEvent" TSRMLS_CC)) {
```

Comment:
Thanks again for your work on this.

This might not be the best differentiator since the ExceptionEvent class was introduced in Symfony 4.3. .


What do you think about if we try something simpler like the following to cover both cases so that we handle it either way:
```
exception = nr_php_call(event, "getThrowable");
if (!nr_php_is_zval_valid_object(exception)) {
    exception = nr_php_call(event, "getException");
  }

```
That way if it is a valid throwable we get that; otherwise, we try for the exception.